### DISCUSSION TITLE
Request for documentation/tutorials on using supabase server-side

### DISCUSSION THREAD

**Discussion by bztravis** [supabase/supabase#29995] [Feature Requests]
Hello, I would really appreciate a tutorial on using supabase server-side (for me, an express backend). 

It's not clear to me in the docs whether using the regular javascript client is acceptable since Supavisor is mentioned but there's no real documentation home for supavisor so I can't tell if it's applicable for my use case or how to set it up.

And in the overview section of auth in the js docs says something like "By default, the supabase client sets persistSession to true and attempts to store the session in local storage. When using the supabase client in an environment that doesn't support local storage, you might notice the following warning message being logged:". It seems like this should apply to me but when using the supabase client in a node/express backend I do not get this error even though localstorage is not available.

It's not clear to me whether these warnings or options apply to me and my scenario or what the ramifications of setting things like 
`autoRefreshToken: false, persistSession: false, detectSessionInUrl: false` are. 

I really appreciate how great the documentation in Supabase is overall and I think that having some on using supabase server-side would be a great addition.

**Comment by GaryAustin1** [comment]
SSR is used for managing sessions with cookies:  https://supabase.com/docs/guides/auth/server-side
This has some express examples:   https://supabase.com/docs/guides/auth/server-side/creating-a-client?queryGroups=environment&environment=client&queryGroups=framework&framework=express 

Supavisor is not involved at all with the REST clients and is involved with direct database access.
